packages/now-static-build/src/frameworks.ts

```diff
@@ -129,6 +129,11 @@ const frameworkList: Framework[] = [
         headers: { 'cache-control': 'max-age=31536000, immutable' },
         continue: true,
       },
+      {
+        src: '^/assets/(images|medias|files)/.*',
+        headers: { 'cache-control': 'max-age=31536000, immutable' },
+        continue: true,
+      },
```

Comment:
You linked to https://v2.docusaurus.io/docs/markdown-features/#assets

However, I don't see where the docs mention the subfolder names or whether the files are hashed like the css/js above.

If they are not hashed, then it can't be cached safely.

Reply:
These paths are deployed by the url-loader with the file containing the hash. Change to a strict regular expression with hashes.